lib/forwardable.rb

```diff
@@ -148,9 +148,8 @@ def instance_delegate(hash)
   #   def_delegator :@records, :map
   #
   def def_instance_delegators(accessor, *methods)
-    methods.delete("__send__")
-    methods.delete("__id__")
-    for method in methods
+    methods.each do |method|
+      next if method.to_s == "__send__" || method.to_s == "__id__"
```

Comment:
I think `next /\A__(?:send|id)__\z/ =~ method` is better because of reducing String objects creation.

Reply:
Sure, that makes sense.